lib/helpers.nix

```diff
@@ -15,6 +15,7 @@ in
   maintainers = import ./maintainers.nix;
   keymaps = import ./keymap-helpers.nix { inherit lib nixvimOptions nixvimTypes; };
   autocmd = import ./autocmd-helpers.nix { inherit lib nixvimOptions nixvimTypes; };
+  deprecation = import ./deprecation-helpers.nix { inherit lib nixvimUtils; };
```

Comment:
I've scoped this for now. Would you rather it be a top-level helper?

Reply:
I think scoped is nicer